styled/auth.js

```diff
@@ -1,33 +1,87 @@
 import styled from 'styled-components/native';
+import Colors from '../assets/Colors';
 
-export const SignUpContainer = styled.View`
+// TODO @tommypoa: Consider creating re-usable containers taking in props for the 3 Auth screens
+
+// StartUpScreen
+
+export const StartUpContainer = styled.View`
   display: flex;
   justify-content: center;
   align-items: center;
   background-color: #fff;
+  margin-top: 241px;
 `;
 
-export const Input = styled.TextInput`
-  width: 350px;
-  height: 55px;
-  background-color: #008550;
-  margin: 10px;
-  padding: 8px;
-  border-radius: 14px;
-  color: white;
-  font-size: 18px;
-  font-weight: 500;
-  align-content: center;
+export const StartUpTitleContainer = styled.View`
+  margin-left: 58px;
+  margin-right: 47px;
+`;
+
+export const StartUpSignInContainer = styled.View`
+  margin-top: 108px;
 `;
 
+export const StartUpLogInContainer = styled.View`
+  margin-top: 16px;
+  margin-left: 42px;
+  margin-right: 47px;
+  width: 286px
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+// LoginScreen
+
 export const LoginContainer = styled.View`
-  flex: 1;
+  margin: 79px 42px 0 42px;
   background-color: #fff;
-  margin-top: 20%;
-  align-content: center;
+`;
+
+// Margin-top value is 140 (figma) - 36 (input margin bottom) = 104px.
+export const LoginButtonContainer = styled.View`
+  margin-top: 104px;
+  margin-bottom: 13px;
```

Comment:
Remove this bottom margin and space this with padding on the Forgot password button so you can standardize this with SignupButtonContainer